Automattic/course
# Fix design, format and Internationalize texts in newsletter, testimonial and mailing list template

Fixes https://github.com/Automattic/course/issues/80

## Changes proposed

- Added internationalization for texts in patterns where it was missing
- Formatted the templates
- Fixed a minor spacing issue in newsletter template (the line on the left of the text was just a bit longer)

## Testing instructions

- Add the Mailing List and Testimonial patterns on a page
- Make sure they look good
- Make sure no texts are missing internationalization in these patterns
- Make sure all texts are rendering

## Screenshots

## Mailing list

Before:
![Screen Shot 2022-11-12 at 12 08 43 AM](https://user-images.githubusercontent.com/6820724/201402334-45c7a694-b673-4bde-82fc-3ca73b96317d.png)


After:
![Screen Shot 2022-11-12 at 12 08 02 AM](https://user-images.githubusercontent.com/6820724/201402355-2b05f431-422e-415e-bc2d-fd6cb3dee299.png)
